On Windows some preprocessing tests fail

On Windows the testcase `testSampleValidates` fails repeatedly. Apparently the reason lies in class `koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest` that uses Unix/Linux platform specific line separator. This is easily fixed, I will propose a pull request.

`Buildfile: E:\swdev\workspaces\ws_default\koopa\build.xml
     [echo]         Please make sure JAVA8_BOOTCLASSES is set to a valid Java 1.8 bootstrap classpath.
     [echo]         You may get builds which are not compatible with Java 1.8 otherwise.
     [echo]     
run-cobol-tests:
    [junit] Running koopa.cobol.grammar.directives.test.FixedDirectivesTests
    [junit] Testsuite: koopa.cobol.grammar.directives.test.FixedDirectivesTests
    [junit] Tests run: 139, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,11 sec
    [junit] Tests run: 139, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,11 sec
    [junit] Running koopa.cobol.grammar.directives.test.FreeDirectivesTests
    [junit] Testsuite: koopa.cobol.grammar.directives.test.FreeDirectivesTests
    [junit] Tests run: 140, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,101 sec
    [junit] Tests run: 140, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,101 sec
    [junit] Running koopa.cobol.sources.test.CobolSourcesValidationTest
    [junit] Testsuite: koopa.cobol.sources.test.CobolSourcesValidationTest
    [junit] Tests run: 15, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,348 sec
    [junit] Tests run: 15, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,348 sec
    [junit] Running koopa.cobol.grammar.preprocessing.test.CobolPreprocessingGrammarTests
    [junit] Testsuite: koopa.cobol.grammar.preprocessing.test.CobolPreprocessingGrammarTests
    [junit] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,045 sec
    [junit] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,045 sec
    [junit] Running koopa.cobol.grammar.test.CobolGrammarTests
    [junit] Testsuite: koopa.cobol.grammar.test.CobolGrammarTests
    [junit] Tests run: 3197, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,858 sec
    [junit] Tests run: 3197, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,858 sec
    [junit] Running koopa.cobol.parser.preprocessing.replacing.test.ReplacingPhraseOperandTest
    [junit] Testsuite: koopa.cobol.parser.preprocessing.replacing.test.ReplacingPhraseOperandTest
    [junit] Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,298 sec
    [junit] Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0,298 sec
    [junit] Running koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest
    [junit] Testsuite: koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest
    [junit] 2026-05-25T07:47:25.001202200Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.014291600Z main ERROR Missing copybook "UNKNOWN" in null
    [junit] 2026-05-25T07:47:25.020579200Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.024586Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.029098300Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.034105Z main ERROR Missing copybook "DUMMY" in null
    [junit] 2026-05-25T07:47:25.039720400Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.045731100Z main ERROR Missing copybook "MOVES" in null
    [junit] Tests run: 19, Failures: 19, Errors: 0, Skipped: 0, Time elapsed: 0,397 sec
    [junit] Tests run: 19, Failures: 19, Errors: 0, Skipped: 0, Time elapsed: 0,397 sec
    [junit] ------------- Standard Output ---------------
    [junit] 2026-05-25T07:47:25.001202200Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.014291600Z main ERROR Missing copybook "UNKNOWN" in null
    [junit] 2026-05-25T07:47:25.020579200Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.024586Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.029098300Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.034105Z main ERROR Missing copybook "DUMMY" in null
    [junit] 2026-05-25T07:47:25.039720400Z main ERROR Missing copybook "MOVES" in null
    [junit] 2026-05-25T07:47:25.045731100Z main ERROR Missing copybook "MOVES" in null
    [junit] ------------- ---------------- ---------------
    [junit] Testcase: testSampleValidates [Copy.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES". GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES". GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyInComment.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  *> COPY "MOVES" REPLACING ==XXX== BY ==FOO==.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". *> COPY "MOVES" REPLACING ==XXX== BY ==FOO==. GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  *> COPY "MOVES" REPLACING ==XXX== BY ==FOO==.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". *> COPY "MOVES" REPLACING ==XXX== BY ==FOO==. GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyMissing.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  COPY "UNKNOWN".
    [junit]  MOVE 33 TO ZZZ.
    [junit] ]> but was:<[ MOVE 11 TO XXX. COPY "UNKNOWN". MOVE 33 TO ZZZ.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  COPY "UNKNOWN".
    [junit]  MOVE 33 TO ZZZ.
    [junit] ]> but was:<[ MOVE 11 TO XXX. COPY "UNKNOWN". MOVE 33 TO ZZZ.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyNonMatchingReplacing.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES" REPLACING ==class== BY ==FOO==. GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES" REPLACING ==class== BY ==FOO==. GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyNoReplacing.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES". GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES". GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyReplacing.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES" REPLACING ==XXX== BY ==FOO==. GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO ZZZ.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "MOVES" REPLACING ==XXX== BY ==FOO==. GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyReplacingDummyOperands.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO FOO-POST.
    [junit] MOVE 22 TO PRE-FOO-POST.
    [junit] MOVE 33 TO PRE-FOO.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "DUMMY" REPLACING ==:TAG:== BY ==FOO==. GO TO END.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  DISPLAY "Start".
    [junit]  MOVE 11 TO FOO-POST.
    [junit] MOVE 22 TO PRE-FOO-POST.
    [junit] MOVE 33 TO PRE-FOO.
    [junit]  GO TO END.
    [junit] ]> but was:<[ DISPLAY "Start". COPY "DUMMY" REPLACING ==:TAG:== BY ==FOO==. GO TO END.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyReplacingx2.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  ONE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO ZZZ.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit]  TWO.
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit]  THREE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit] ]> but was:<[ ONE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==YYY== BY ==FUM==. ADD 1 TO XXX YYY ZZZ. TWO. COPY "MOVES" REPLACING ==YYY== BY ==FUM== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ. THREE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  ONE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO ZZZ.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit]  TWO.
    [junit]  MOVE 11 TO XXX.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit]  THREE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO YYY.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit] ]> but was:<[ ONE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==YYY== BY ==FUM==. ADD 1 TO XXX YYY ZZZ. TWO. COPY "MOVES" REPLACING ==YYY== BY ==FUM== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ. THREE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [CopyReplacingx3.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  ONE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit] ]> but was:<[ ONE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==YYY== BY ==FUM== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  ONE.
    [junit]  MOVE 11 TO FOO.
    [junit] MOVE 22 TO FUM.
    [junit] MOVE 33 TO FEE.
    [junit]  ADD 1 TO XXX YYY ZZZ.
    [junit] ]> but was:<[ ONE. COPY "MOVES" REPLACING ==XXX== BY ==FOO== ==YYY== BY ==FUM== ==ZZZ== BY ==FEE==. ADD 1 TO XXX YYY ZZZ.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [NoPreprocessing.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  MOVE 22 TO YYY.
    [junit]  MOVE 33 TO ZZZ.
    [junit] ]> but was:<[ MOVE 11 TO XXX. MOVE 22 TO YYY. MOVE 33 TO ZZZ.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  MOVE 22 TO YYY.
    [junit]  MOVE 33 TO ZZZ.
    [junit] ]> but was:<[ MOVE 11 TO XXX. MOVE 22 TO YYY. MOVE 33 TO ZZZ.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceAlso.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FEE TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ALSO ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE ALSO ==FOO== BY ==FEE==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FEE TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ALSO ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE ALSO ==FOO== BY ==FEE==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceAlsoFirst.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceAlsoLastOff.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FEE TO FUM.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ALSO ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE ALSO ==FOO== BY ==FEE==. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FEE TO FUM.
    [junit]  
    [junit]  MOVE FOO TO FUM.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ALSO ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE ALSO ==FOO== BY ==FEE==. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY. REPLACE LAST OFF. MOVE FOO TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceLastOffNoOn.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE XXX TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE XXX TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceOffNoOn.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE XXX TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE XXX TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceOn.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  
    [junit]  MOVE 22 TO FOO.
    [junit]  MOVE 33 TO FOO.
    [junit] ]> but was:<[ MOVE 11 TO XXX.  MOVE 22 TO FOO. MOVE 33 TO FOO.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  
    [junit]  MOVE 22 TO FOO.
    [junit]  MOVE 33 TO FOO.
    [junit] ]> but was:<[ MOVE 11 TO XXX.  MOVE 22 TO FOO. MOVE 33 TO FOO.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceOnOff.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  
    [junit]  MOVE 22 TO FOO.
    [junit]  
    [junit]  MOVE 33 TO XXX.
    [junit] ]> but was:<[ MOVE 11 TO XXX.  MOVE 22 TO FOO. REPLACE OFF. MOVE 33 TO FOO.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE 11 TO XXX.
    [junit]  
    [junit]  MOVE 22 TO FOO.
    [junit]  
    [junit]  MOVE 33 TO XXX.
    [junit] ]> but was:<[ MOVE 11 TO XXX.  MOVE 22 TO FOO. REPLACE OFF. MOVE 33 TO FOO.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceOnOffx2.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY. REPLACE ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE XXX TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY. REPLACE ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)
    [junit] Testcase: testSampleValidates [ReplaceOverrides.ppsample](koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest):	FAILED
    [junit] expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] junit.framework.AssertionFailedError: expected:<[
    [junit]  MOVE XXX TO YYY.
    [junit]  
    [junit]  MOVE FOO TO YYY.
    [junit]  
    [junit]  MOVE XXX TO FUM.
    [junit]  
    [junit]  MOVE XXX TO YYY.
    [junit] ]> but was:<[ MOVE XXX TO YYY.  MOVE FOO TO YYY. REPLACE ==YYY== BY ==FUM==. MOVE FOO TO YYY. REPLACE OFF. MOVE FOO TO YYY.]>
    [junit] 	at koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest.testSampleValidates(PreprocessingSourceTest.java:68)
    [junit] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    [junit] 	at koopa.core.util.test.Files$FileRunner$1.evaluate(Files.java:94)
    [junit] 	at koopa.core.util.test.Files.run(Files.java:58)

BUILD FAILED
E:\swdev\workspaces\ws_default\koopa\build.xml:358: Test koopa.cobol.parser.preprocessing.test.PreprocessingSourceTest failed

Total time: 6 seconds
`

--- a/test/cobol/koopa/cobol/parser/preprocessing/test/PreprocessingSourceTest.java
+++ b/test/cobol/koopa/cobol/parser/preprocessing/test/PreprocessingSourceTest.java
@@ -25,6 +25,7 @@ public class PreprocessingSourceTest implements FileBasedTest {
 
 	private static final String INPUT_PREFIX = "<";
 	private static final String EXPECTED_PREFIX = ">";
+	private static final String LINE_SEPARATOR = System.lineSeparator();
 
 	@Override
 	public File[] getFiles() {
@@ -79,11 +80,11 @@ public class PreprocessingSourceTest implements FileBasedTest {
 				while ((line = br.readLine()) != null) {
 					if (line.startsWith(INPUT_PREFIX)) {
 						input.append(line.substring(INPUT_PREFIX.length()));
-						input.append('\n');
+						input.append(LINE_SEPARATOR);
 
 					} else if (line.startsWith(EXPECTED_PREFIX)) {
 						expected.append(line.substring(EXPECTED_PREFIX.length()));
-						expected.append('\n');
+						expected.append(LINE_SEPARATOR);
 					}
 				}
 

